tools/grid/test/grid.test.ts

```diff
@@ -31,6 +31,12 @@ describe('Grid', () => {
                 <div>${Default()}</div>
             `)
     );
+    beforeEach(() => {
```

Comment:
@majornista can you add some a11y notes for this in `packages/grid/README.md`?

Reply:
See my comment here: https://github.com/adobe/spectrum-web-components/pull/5171/files#r2014426185